openpype/hosts/traypublisher/plugins/publish/collect_frame_data_from_asset_entity.py

```diff
@@ -23,22 +23,16 @@ class CollectMissingFrameDataFromAssetEntity(
     def process(self, instance):
         if not self.is_active(instance.data):
             return
-        missing_keys = []
+
+        asset_data = instance.data["assetEntity"]["data"]
+
         for key in (
             "fps",
             "frameStart",
             "frameEnd",
             "handleStart",
             "handleEnd"
         ):
-            if key not in instance.data:
-                missing_keys.append(key)
-        keys_set = []
-        for key in missing_keys:
-            asset_data = instance.data["assetEntity"]["data"]
-            if key in asset_data:
-                instance.data[key] = asset_data[key]
-                keys_set.append(key)
-        if keys_set:
-            self.log.debug(f"Frame range data {keys_set} "
-                           "has been collected from asset entity.")
+            instance.data[key] = asset_data[key]
```

Comment:
You should not override the values if are already filled, and you should not expect thay are filled on asset data. As far as I remember, during editorial are all of these values prefilled from completelly different source and should not be change by asset entity, which may not exist yet.